packages/botonic-plugin-ner/src/model/model-info.ts

```diff
@@ -0,0 +1,28 @@
+import { NerConfig } from '@botonic/nlp/dist/tasks/ner/storage/types'
+import { Locale } from '@botonic/nlp/dist/types'
+import { LayersModel, loadLayersModel } from '@tensorflow/tfjs'
+import axios, { AxiosPromise } from 'axios'
+
+import { CONFIG_FILENAME, MODEL_FILENAME } from '../constants'
+
+export class ModelInfo {
+  private config: AxiosPromise<NerConfig>
+  private model: Promise<LayersModel>
+
+  constructor(readonly locale: Locale, readonly uri: string) {
+    this.init()
+  }
+
+  private init() {
```

Comment:
what is the purpose of init? drop it 
actually it causes a problem, since in strict mode the compiler will complain that config & model are not initialized in the constructor